jest.config.js

```diff
@@ -15,4 +15,5 @@ module.exports = {
   moduleNameMapper: pathsToModuleNameMapper(tsconfig.compilerOptions.paths, { prefix: `${ROOT_DIR}/` }),
   collectCoverage: false,
   cacheDirectory: resolve(ROOT_DIR, `${CI ? '' : 'node_modules/'}.cache/jest`),
+  preset: 'ts-jest',
```

Comment:
I avoided to use ts-jest in purpose to have faster compilation and reduce overhead of tsc. I don't think we need type checking in tests. At least, it is not CI's job I believe. So it's done by babel already.

Reply:
I would agree that for most cases the CI should not have to run type checking. But for open source tools and libraries with advanced generic types like the `ResolversComposerMapping` type, they should absolutely be tested . They are still a written piece of code and should be treated as such since they are prone to errors like any other code. 

Tsc and babel only validate that types are correct. They don't know how to test intended behavior within generic types and ultimately allows code like this to pass type checking

```typescript
interface Resolvers {
  Query?: {
    foo: () => string
  }
}

export const rcMap: ResolversComposerMapping<Resolvers> = {
  Query: {
    foo: 'hello'
  }
}
```

I understand that faster compile times and less overhead is desired. But if it compromises type robustness then it's not a great trade-off and makes using said types superfluous since they might or might not work as intended.